src/app/system/manage-data-tables/view-data-table/view-data-table.component.html

```diff
@@ -0,0 +1,61 @@
+<div class="container m-b-20" fxLayout="row" fxLayoutAlign="end" fxLayoutGap="20px">
+
+    <button mat-raised-button color="primary" [routerLink]="['../../edit', dataTableData.registeredTableName]" >
+        <fa-icon icon="edit"></fa-icon>&nbsp;&nbsp;
+        Edit
+    </button>
+
+    <button mat-raised-button color="warn" (click)="delete()">
+        <fa-icon icon="trash"></fa-icon>&nbsp;&nbsp;
+        Delete
+    </button>
+
+</div>
+
+<div class="container m-b-15" fxLayoutAlign="start">
+    <mat-card>
+        <mat-card-subtitle>Associated With {{ dataTableData.applicationTableName }}</mat-card-subtitle>
+    </mat-card>
+</div>
+
+<div class="container">
+
+    <div class="mat-elevation-z8">
+
+        <table mat-table [dataSource]="dataSource" matSort>
+
+            <ng-container matColumnDef="columnName">
+                <th mat-header-cell *matHeaderCellDef mat-sort-header> Field Name </th>
+                <td mat-cell *matCellDef="let dataTable"> {{ dataTable.columnName }} </td>
+            </ng-container>
+
+            <ng-container matColumnDef="columnType">
```

Comment:
Should be similar to the property name.

Reply:
Sorry, I couldn't get you.

Reply:
`columnDisplayType` is the property name and you have kept column definition name as `columnType` instead. It should be the same unless the object is nested in some cases.